client/src/components/columns.jsx

```diff
@@ -1,55 +1,38 @@
 import React, { Component } from "react";
 import axios from "axios";
-let arrhappy = [];
```

Comment:
in react it's not possible to keep any state outside of state without not expected problems